kubernetes-deployment/massdriver.yaml

```diff
@@ -5,32 +5,53 @@ ref: github.com/YOUR_ORG/<md .Name md>
 access: private
 type: application
 
+app:
+  envs:
+    # Use jq expressions to build environment variables from input params or connections
+    LOG_LEVEL: .params.log_level
+    # MONGO_DSN: .connections.mongo_authentication.data.authentication.username + ":" + .connections.mongo_authentication.data.authentication.password + "@" + .connections.mongo_authentication.data.authentication.hostname + ":" + (.connections.mongo_authentication.data.authentication.port|tostring)
+  policies: []
+    # Use jq expressions to select policies from artifact security blocks
+    # - .connections.sqs.data.security.iam.subscribe
+
 params:
   properties:
-    name:
-      type: string
     namespace:
+      title: Kubernetes Namespace
       type: string
       default: default
     image:
+      title: Container Image
       type: object
       properties:
         repository:
-          type: string
-        sha:
+          title: Repository
+          description: "Container image repository to use for this application. See: https://app.massdriver.cloud/container-repositories"
           type: string
         tag:
+          title: Tag
+          description:
           type: string
-    resource_requests:
+          minLength: 1
+    resourceRequests:
       type: object
       properties:
         cpu:
+          title: CPU
+          description: The minimum CPU resources required for this application.
           type: string
+          # TODO: use custom rjsf widget here
```

Comment:
@WillBeebe @chrisghill what format is this expecting? The k8s serialization format? Our RJSF doesn't support that, right?

I'm thinking use the unit RJSF widget as is and for CPU just use decimal and we can convert to millicpu... 0.5 -> 500m 1 -> 1000m etc.

For memory, maybe set the units as `Mi` and let them enter whatever (MB, gb, gib) but always serialize out to k8s serialization

Reply:
Yes and yes, it's a straight passthrough so whatever works on k8s.

This is a great place to use the unit widget. 👍